packages/dds/tree/src/simple-tree/core/treeNodeSchema.ts

```diff
@@ -168,6 +167,23 @@ export interface TreeNodeSchemaCore<
 	 * Setting this to false adjusts the insertable types to disallow cases which could be impacted by these inconsistencies.
 	 */
 	readonly implicitlyConstructable: ImplicitlyConstructable;
+
+	/**
+	 * Schema children of this node can have.
```

Comment:
```suggestion
	 * Schema children that this node can have.
```

Reply:
Is this just the types that _direct_ children can have? Or does it include all descendent types?

Reply:
Phrasing here is a bit ambiguous but "children of this node" is the noun phrase here not "Schema children". And yes, its transitive not just direct. I'll try and clarify